nodejs/week1/meals/src/backend/routes/cheap-meals.js

```diff
@@ -0,0 +1,17 @@
+const express = require("express");
+const app = express();
```

Comment:
This is not needed, the idea is to export the router function and register it with a route in index.js. The nice way about having all the mappings between route => routers in one file is that you can see at a glance all routes that are being handled by a given router. Then if there's some specific router that has a problem or which you want to expand you can jump into that one.